typhon/cli.py

```diff
@@ -54,48 +54,65 @@ def typhon_cli(args):
         print(f'Typhon: Version {typhon.__version__} from {typhon.__file__}')
         return
 
-    try:
-        import happi
-    except (ImportError, ModuleNotFoundError):
-        logger.exception("Unable to import happi to load devices!")
-        return
-
-    logger.debug("Creating Happi Client ...")
-    client = happi.Client.from_config(cfg=args.happi_cfg)
-
-    logger.debug("Creating widgets ...")
-    app = QApplication.instance()
+    # Deal with stylesheet
+    logger.debug("Applying stylesheet ...")
```

Comment:
This debug message seems misplaced, since the next thing done here is create the app rather than call `use_stylesheet`